packages/jupyter-ai/jupyter_ai/models.py

```diff
@@ -3,6 +3,9 @@
 from jupyter_ai_magics.providers import AuthStrategy, Field
 from pydantic import BaseModel
 
+DEFAULT_CHUNK_SIZE = 2000
+DEFAULT_CHUNK_OVERLAP = 100
+
```

Comment:
The default values are actually never sourced from here; these values, if unspecified, are actually set by the default arguments of `LearnChatHandler#learn_dir()`, since those are resolved first before creating the `IndexedDir` object.

I think it'd be better to move these constants back to `learn.py` and use these as the default arguments instead of hard-coding them (e.g. `chunk_size: int = DEFAULT_CHUNK_SIZE`). Then you can feel free to drop the defaults on the model here in this file, since `chunk_size` and `chunk_overlap` will always be defined.

Reply:
I kept these in the model for compatibility with the existing /learn metadata which will probably break without these values in the model.

Reply:
Ah, good call.